settings.gradle

```diff
@@ -1 +1 @@
-rootProject.name = 'Fragment Fingerprints'
\ No newline at end of file
+rootProject.name = 'FragmentFingerprinter'
```

Comment:
What did you do that for? Asked mainly out of curiosity.

Reply:
I had a problem creating the jar file. It was complaining about a space. So I just removed it.